subseas_flex_fcst/maproom.py

```diff
@@ -45,26 +48,75 @@
 
 APP.layout = layout.app_layout
 
-
-def read_cptdataset(y_transform=False):
-
-    fcst_mu = cptio.open_cptdataset(Path(DATA_PATH, Path(CONFIG["forecast_mu_file"])))
+#function to open all datasets and then combine them
+def combine_cptdataset(dataDir,filePattern,dof=False,y_transform=False):
```

Comment:
`y_transform` argument is never used?

Reply:
@remicousin

Reply:
But the variable is never used in this function. It could be eliminated without changing the behavior.

Reply:
that is true. read_cptdataset deals with it.

Reply:
So I will remove `y_transform` from the `combine_cptdataset` function.

Then if I am understanding correctly @remicousin , there are cases in this app where `y_transform` both needs to be false and true? if that is the case then we would not bother to have it in the config file?

Reply:
No, for a given app/set-of-files, y_transform is either true or false throughout. To manipulate distributions, we need mean and standard deviation. The standard deviation of the forecast comes from the var (variance) file. However, that variable has sometimes been through a so-called Y-transform. If no Y-transform (false), we can use the var file. If Y-transform (true), the var file is not the variance we expect and we don't know how to use it or transform it back to what we need. But we can then estimate that variance (or standard deviation) using the hindcast and obs. So the y_transform boolean triggers using that or the other.